src/screens/internal/CompareSnapshots/SnapshotVersions.tsx

```diff
@@ -47,12 +47,20 @@ export function useSnapshotVersion(
 ): SnapshotVersion | null {
   const [version, setVersion] = useState<SnapshotVersion | null>(null);
   useEffect(() => {
-    setVersion(null);
-    if (snapshot !== null) {
-      new Api(snapshotUrl(snapshot)).fetchVersionInfo().then(version => {
-        setVersion(version);
-      });
+    async function fetchVersion() {
```

Comment:
Converting this to an async function so that the try/catch will execute correctly